### DISCUSSION TITLE
HELP !!!   I lost my ADMIN Credentials for my DSJAS install.

### DISCUSSION THREAD

**Discussion by ScammerStopperZ** [DSJAS/DSJAS#68] [General Q&A] [answered]
I installed DSJAS, got it working, and used it.  I somehow lost my [very secure] Admin password.   How can I get in to the MySQL DB to reset that password as well as work with the existing ScamBait accounts? 
Thanks in advance for the help !! 
Jim

**Comment by ejv2** [comment]
How is the database installed? Is it local under xampp or mamp or similar?

**Comment by ejv2** [comment]
Hello @ScammerStopperZ, is this issue resolved?

**Comment by ScammerStopperZ** [comment, marked as answer]
Apologies, I did figure out what I was doing wrong, and neglected to update
the ticket. Again, my apologies. Thank you for everything you do on this
product. Greatly appreciated. Hoping you can integrate support for things
like Zelle, Bitcoin, PayPal, etc, so it becomes an even greater scam
baiting product.

**Reply by ejv2** [reply]
No problem at all. I am just trying to go through the issues backlog to close off some old tickets. If this issue is resolved, I will go ahead and mark this as done. If you think of any other suggestions, please feel free to open up an issue in the tracker and I will get to it once I have the chance.

**Comment by ejv2** [comment]
Closing as resolved
